collective/disqus/browser/disqus_summary_view.py

```diff
@@ -22,7 +22,7 @@ def get_counter_js(self):
 
         if short_name:
             result = """
-<script type="text/javascript" async="async" src="http://{0}.disqus.com/count.js">
+<script type="text/javascript" async="async" src="https://{0}.disqus.com/count.js">
```

Comment:
HTML5 specification urges authors to omit the attribute rather than provide a redundant MIME type; do you mind to remove the `type` attribute also? instead we can use:

```xml
<script async src="https://{0}.disqus.com/count.js">
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script